module/scheduler_components/deterministic_lock_manager.h

```diff
@@ -20,12 +28,17 @@ using std::vector;
 
 namespace slog {
 
+using KeyReplica = string;
+
+enum class AcquireLocksResult {ACQUIRED, WAITING, ABORT};
```

Comment:
This implementation doesn't use aborts here, but I think we could leave it in for future optimizations